sa/_db/migrations/20151003141825_AddKeyAuthorization.sql

```diff
@@ -0,0 +1,12 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+ALTER TABLE `challenges` ADD COLUMN (`keyAuthorization` mediumblob);
```

Comment:
Is this really bigger than `2^16` (the size of `blob`)? Does it need to support 16 MB/authorization?